lib/apiservers/portlayer/swagger.json

```diff
@@ -59,6 +59,115 @@
 				}
 			}
 		},
+		"/kv": {
+			"get": {
+				"description": "Gets value from k/v store",
+				"tags": [
+					"kv"
+				],
+				"operationId": "GetValue",
+				"consumes": [
+					"application/json"
+				],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"name": "key",
+						"type": "string",
+						"in": "query",
+						"required": true
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/KeyValue"
+						}
+					},
+					"404": {
+						"description": "Not found"
+					},
+					"default": {
+						"description": "error",
+						"schema": {
+							"$ref": "#/definitions/Error"
+						}
+					}
+				}
+			},
+			"delete": {
+				"description": "deletes entry in k/v store",
+				"tags": [
+					"kv"
+				],
+				"operationId": "DeleteValue",
+				"consumes": [
+					"application/json"
+				],
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+					"name": "key",
+					"type": "string",
+					"in": "query",
+					"required": true
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK"
+					},
+					"404": {
+						"description": "Not found"
+					},
+					"default": {
```

Comment:
@cgtexmex The default can still come back with other errors besides the one you are returning. It also covers other swagger related errors. It is better to create your own error type and return that one, that way if swagger has some error you will get a generic default(or a no default response type error idr which). If we do it this way we avoid having to inspect the type, model assignment, and error code to make sure we got an error from the portlayer and not the middleware between the portlayer and the personality.